src/tree/AzureDBAPIStep.ts

```diff
@@ -51,7 +52,8 @@ export class AzureDBAPIStep extends AzureWizardPromptStep<IPostgresServerWizardC
             ];
         } else {
             promptSteps = [
-                new CosmosDBAccountNameStep()
+                new CosmosDBAccountCapacityStep(),
```

Comment:
It's odd that the capacity mode step comes before the accountNameStep.

Reply:
accountNameStep can go before CapacityStep, it shouldn't cause a problem. 

I was going for this setup:

1. "Select an Azure Database Server."
2. and pick the next categorization which is capacity model with "Select a capacity model" prompt.
3. and then start creating the server starting with the nameStep.